kylieanyce/juice-analytics-project
# attempting to return json data in json file

# Description
Trying to get the json data from the api to output into a json file.

## Type of change


# Testing Instructions
```
git checkout main
git fetch --all
git checkout kb-juiceCount
```
Run the server and you will see how it doesn't automatically fill the .json file. I think I am changing the response into JSON data twice and need to remove lines 19 & 20.

# Checklist:

- [/] I have added test instructions that prove my fix is effective or that my feature works
